pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -149,6 +149,9 @@ func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfigura
 		// Keep the same as default NodeStatusUpdateFrequency
 		obj.CPUManagerReconcilePeriod = metav1.Duration{Duration: 10 * time.Second}
 	}
+	if obj.TopologyManagerPolicy == "" {
+		obj.TopologyManagerPolicy = "none"
```

Comment:
nit, define constants for the allowed values (in the same file as the config type) and use them here, in validation, and in fuzzing